web/styles/left_sidebar.css

```diff
@@ -1099,6 +1099,7 @@ li.topic-list-item {
     }
 }
 
+#direct_messages_sticky_header.zoom-out,
 .zoom-out {
```

Comment:
This is a weird set of selectors; why do we need the duplicate with extra specificity?

Reply:
Added a comment:

```
/* Since direct-messages-sticky-header also has the
`input-append` class accompanying it. The display property
of that class will overwrite display: none if we don't have
a more specific CSS rule. It will also overwrite `display: none`
even if `.zoom-out` properties are delcared after the `.input-append`
properties since the latter is more specific.  */
```

Reply:
Great. As a tiny nit, this comment doesn't follow our conventions for CSS comments (secondary lines should have two extra spaces); I'll fix that up if I can merge today.